### PR TITLE
Aspec ratio resizer

### PR DESCRIPTION
**Pull request by antoineraoulimpargo**
Related issue: https://github.com/impargo/impargo-apps/issues/4536

### CODE REVIEW

File: example/app.jsx [hunk 1]
```diff
@@ -85,7 +85,10 @@ class App extends React.Component {
               cropRadius={50}
               onCrop={this.onCrop}
               onClose={this.onClose}
+              lineWidth={5}
               src={this.state.src}
+              changeAspecRatio
```
Comment (ghost) [inline example/app.jsx:90]: Minor: We can also remove the `width: '150px'` styling from the `img` tag so that the preview is not skewed

**Review by ghost — APPROVED** [review]
LGTM :shipit: 

Just one minor comment mentioned in the review.

Also one minor improvement that could be made:
When hovering over the sides its a bit hard to get the resize arrows to show (you have to be precise with your mouse)

File: src/avatar.jsx [hunk 2]
```diff
@@ -273,37 +283,61 @@ class Avatar extends React.Component {
     const isBottomCorner = scale => crop.y() + scaledRadius(scale)/2 > stage.height();
     const calcBottom = () => stage.height() - crop.height()/2 - 1;
     const isNotOutOfScale = scale => !isLeftCorner(scale) && !isRightCorner(scale) && !isBottomCorner(scale) && !isTopCorner(scale);
+    const isWithinAspecRatio = (aspecRatio) => {
+      console.log(aspecRatio)
```
Comment (Ahmed-Shaarawy) [inline src/avatar.jsx]: Minor: I think we can remove this `console.log()`.

**Review by Ahmed-Shaarawy — APPROVED** [review]

**Comment by antoineraoulimpargo** [comment]
@SakrImpargo Increasing the threshold and border width did fix your comment.
wdyt ?

**Comment by ghost** [comment]
Perfect

**Pull request merged**